services/alerts/src/makeAlertsManager.ts

```diff
@@ -1,36 +1,39 @@
-import { Alert, AlertsManager, AlertsManagerAlert } from './types'
+import type { Alert, AlertRef, AlertsManager, AlertsMap } from './types'
 
-type SetAlertsFunction = React.Dispatch<
-    React.SetStateAction<AlertsManagerAlert[]>
->
-
-const createAlertManagerAlert = (
-    alert: Alert,
-    id: number,
-    remove: (id: number) => void
-): AlertsManagerAlert => ({
-    ...alert,
-    id,
-    remove: () => remove(id),
-})
+const toVisibleAlertsArray = (alertsMap: AlertsMap) =>
+    Array.from(alertsMap.values()).sort(
+        (a, b) => (a.id as number) - (b.id as number)
+    )
```

Comment:
As a map is iterable based on insertion order ([ref](https://hackinbits.com/articles/how-to-iterate-a-map-in-javascript---map-part-2)) I'm not sure this sort is necessary?